src/main/java/at/ac/tuwien/kr/alpha/solver/DefaultSolver.java

```diff
@@ -88,17 +90,16 @@ public DefaultSolver(AtomStore atomStore, Grounder grounder, NoGoodStore store,
 				BranchingHeuristicFactory.getInstance(branchingHeuristic, grounder, assignment, choiceManager, random),
 				new NaiveHeuristic(choiceManager));
 		this.disableJustifications = disableJustifications;
+		this.performanceLog = new PerformanceLog(choiceManager, (TrailAssignment) assignment, 1000l);
```

Comment:
Why is `msBetweenOutputs` initialized with `10001` here?

Reply:
It is not initialized with `10001` but with `1000l`, which preserves the behaviour we had before (one log output every 1000 ms). Admittedly, the two numbers are hard to distinguish visually. For some reason I had thought I needed to provide a `long` explicitly, which is not the case. The `l` has now been dropped.